e2e-test/cypress/integration/project_assignee_spec.js

```diff
@@ -73,6 +73,35 @@ describe("Project Assignee", function() {
     });
   }
 
+  function showAssignProjectDropdown(projectName) {
+    cy.createProject(projectName, "project close test").then(({ id }) => {
+      projectId = id;
+      cy.visit("/projects/" + projectId);
+      permissionsBeforeTesting = { project: {} };
+      cy.listProjectPermissions(projectId).then(permissions => {
+        permissionsBeforeTesting.project = permissions;
+      });
+
+      setupConfirmationDialog()
+      cy.get("@firstUncheckedRadioButton").then(firstUncheckedRadioButton => {
+      cy.get(firstUncheckedRadioButton)
+        .should("not.be.checked")
+        .check();
+      });
+    });
+  }
+
+  it("Assign project should throw error due to bad payload, the display character length less than three", function() {
+    const projectName = "p";
+    showAssignProjectDropdown(projectName);
+  });
```

Comment:
you do not check if an error is thrown. Furthermore, there is no error when assigning a project which name only has one charactar.